ai_database.py

```diff
@@ -12,54 +12,68 @@ def load_config():
     return config
 
 config = load_config()
+chat_history_enabled = config['chat_history']
+summary_enabled = config['summary']
+branching_enabled = config['branching']
+conversations_enabled = config['conversations']
 
 
 model = config['ai_model']
-module_name = f"{model.replace('.', '_')}_module"
-ai_module = importlib.import_module(f"{module_name}")
-
+model_module_name = f"{model.replace('.', '_')}_module"
+# the location of the module is now in a subfolder \modules\ai_model\
+ai_module = importlib.import_module(f"modules.ai_model.{model_module_name}")
 
 gui = config['gui']
-module_name = f"{gui.replace('.', '_')}_gui"
-gui_module = importlib.import_module(f"{module_name}")
+gui_module_name = f"{gui.replace('.', '_')}_gui"
+gui_module = importlib.import_module(f"{gui_module_name}")
 
 
 database = config['database']
-module_name = f"{database.replace('.', '_')}_database"
-database_module = importlib.import_module(f"{module_name}")
+database_module_name = f"{database.replace('.', '_')}_database"
+database_module = importlib.import_module(f"modules.database.{database_module_name}")
```

Comment:
Could we take this directly from the config?

Reply:
yes we can. i added a pull request for the changes due to it modifying multiple files